bridge/engine/EngineMixer.cpp

```diff
@@ -3190,4 +3192,100 @@ void EngineMixer::processRecordingMissingPackets(const uint64_t timestamp)
     }
 }
 
+class SetMaxMediaBitrateJob : public jobmanager::CountedJob
+{
+public:
+    SetMaxMediaBitrateJob(transport::RtcTransport& transport,
+        uint32_t reporterSsrc,
+        uint32_t ssrc,
+        uint32_t bitrateKbps,
+        memory::PacketPoolAllocator& allocator)
+        : CountedJob(transport.getJobCounter()),
+          _transport(transport),
+          _ssrc(ssrc),
+          _reporterSsrc(reporterSsrc),
+          _bitrate(bitrateKbps),
+          _allocator(allocator)
+    {
+    }
+
+    void run() override
+    {
+        auto* packet = memory::makePacket(_allocator);
+        if (!packet)
+        {
+            return;
+        }
+
+        auto& tmmbr = rtp::RtcpTemporaryMaxMediaBitrate::create(packet->get(), _reporterSsrc);
+        tmmbr.addEntry(_ssrc, _bitrate * 1000, 34);
+        packet->setLength(tmmbr.size());
+        _transport.protectAndSend(packet, _allocator);
+    }
+
+private:
+    transport::RtcTransport& _transport;
+    const uint32_t _ssrc;
+    const uint32_t _reporterSsrc;
+    const uint32_t _bitrate;
+    memory::PacketPoolAllocator& _allocator;
+};
+
+void EngineMixer::checkVideoBandwidth(const uint64_t timestamp)
+{
+    if (utils::Time::diffGE(_lastVideoBandwidthCheck, timestamp, utils::Time::sec * 3))
```

Comment:
Could you invert this with return instead, to avoid having everything in the function in an extra scope?